Python/text_based_RPG_game.py

```
@@ -0,0 +1,475 @@
+                print(f"💥 You deal {damage} damage to {enemy.name}!")
+        
+        elif action == '2':  # Special Ability
+            if not player.use_special_ability(enemy):
+                continue  # Invalid special, try again
+        
+        elif action == '3':  # Use Item
+            if not use_item_in_combat(player):
+                continue  # No valid item used, try again
+        
+        elif action == '4':  # Flee
+            if random.random() < 0.6:  # 60% chance to flee
```

Ruff (0.14.1)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)